server/src/Dao/RevisionDao.php

```diff
@@ -9,7 +9,7 @@ class RevisionDao extends AbstractDao {
 	/**
 	 * Fetch user interaction in commonly edited pages
 	 *
-	 * @param int[] $users
+	 * @param array $users List of usernames/IP(s), ids, or mixed
```

Comment:
I don't think this supports `mixed`?
See the docblock on `getUserSearchColum()`